docs/source/securing-calico.rst

```diff
@@ -0,0 +1,84 @@
+.. # Copyright (c) Metaswitch Networks 2015. All rights reserved.
+   #
+   #    Licensed under the Apache License, Version 2.0 (the "License"); you may
+   #    not use this file except in compliance with the License. You may obtain
+   #    a copy of the License at
+   #
+   #         http://www.apache.org/licenses/LICENSE-2.0
+   #
+   #    Unless required by applicable law or agreed to in writing, software
+   #    distributed under the License is distributed on an "AS IS" BASIS,
+   #    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+   #    implied. See the License for the specific language governing
+   #    permissions and limitations under the License.
+
+Securing Calico
+===============
+
+What Calico does and does not provide
+-------------------------------------
+
+Currently, Calico implements security policy that ensures that:
+
+- (assuming its host is not compromised) an endpoint cannot spoof its source
```

Comment:
I don't think 'assuming its host is not compromised' is necessary here. Almost all of these guarantees assume the host is not compromised. It may be better to expand on exactly what you lose if a host is compromised.